src/features/devices/models/device-additions.ts

```diff
@@ -59,8 +59,13 @@ export const addToModel = (
 	// so it's still provisioning.
 	const isPreProvisioning: AndNode = [
 		'And',
-		isOverallOffline,
+		['Equals', ['ReferencedField', 'device', 'is online'], ['Boolean', false]],
```

Comment:
Is this redundant since having any connectivity event info would be the real tell?

Reply:
It should probably be fine without it. I only really added it for dependent devices.